File: Plugins/Aspose_Email_Java_for_PHP/src/aspose/email/ProgrammingEmail/CreateNewEmail.php
<?php
namespace Aspose\Email\ProgrammingEmail;

use com\aspose\email\MailMessage as MailMessage;
use com\aspose\email\MailAddress as MailAddress;
use com\aspose\email\MailMessageSaveType as MailMessageSaveType;

class CreateNewEmail{

    public static function run($dataDir=null){

        # Create a new instance of MailMessage class
        $message = new MailMessage();

        # Set subject of the message
        $message->setSubject("New message created by Aspose.Email for Java");

        $mail_address = new MailAddress();

        # Set Html body
        $message->setHtmlBody("<b>This line is in bold.</b> <br/> <br/>" .
            "<font color=blue>This line is in blue color</font>");

        # Set sender information
        $message->setFrom(new MailAddress("from@domain.com", "Sender Name", false));

        # Add TO recipients
        $message->getTo()->add(new MailAddress("to1@domain.com", "Recipient 1", false));
        $message->getTo()->add(new MailAddress("to2@domain.com", "Recipient 2", false));

        # Add CC recipients
        $message->getCC()->add(new MailAddress("cc1@domain.com", "Recipient 3", false));
        $message->getCC()->add(new MailAddress("cc2@domain.com", "Recipient 4", false));

        # Save message in EML and MSG formats
        $mail_message_save_type = new MailMessageSaveType();
        $message->save($dataDir . "Message.eml", $mail_message_save_type->getEmlFormat());
        $message->save($dataDir . "Message.msg", $mail_message_save_type->getOutlookMessageFormat());
        # Display Status
        print "Created email messages Successfully.".PHP_EOL;
    }
}
?>

File: Plugins/Aspose-Email-Java-for-Jython/asposeemail/ProgrammingEmail/CreateNewEmail.py
from asposeemail import Settings
from com.aspose.email import MailMessage
from com.aspose.email import MailAddress
from com.aspose.email import MailMessageSaveType

class CreateNewEmail:

    def __init__(self):
                
        dataDir = Settings.dataDir + 'ProgrammingEmail/CreateNewEmail/'
        
        # Create a instance of MailMessage class
        message = MailMessage()

        # Set subject of the message
        message.setSubject("New message created by Aspose.Email for Java")

        mail_address = MailAddress

        # Set Html body
        message.setHtmlBody("<b>This line is in bold.</b> <br/> <br/>" +
            "<font color=blue>This line is in blue color</font>")

        # Set sender information
        message.setFrom(MailAddress("from@domain.com", "Sender Name", False))

        # Add TO recipients
        message.getTo().add(MailAddress("to1@domain.com", "Recipient 1", False))
        message.getTo().add(MailAddress("to2@domain.com", "Recipient 2", False))

        # Add CC recipients
        message.getCC().add(MailAddress("cc1@domain.com", "Recipient 3", False))
        message.getCC().add(MailAddress("cc2@domain.com", "Recipient 4", False))

        # Save message in EML and MSG formats
        mail_message_save_type = MailMessageSaveType()
        message.save(dataDir + "Message.eml", mail_message_save_type.getEmlFormat())
        message.save(dataDir + "Message.msg", mail_message_save_type.getOutlookMessageFormat())
        # Display Status
        print "Created email messages Successfully."
 
if __name__ == '__main__':        
    CreateNewEmail()